java/src/org/certificatetransparency/ctlog/MerkleTreeLeaf.java

```diff
@@ -0,0 +1,16 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class MerkleTreeLeaf {
```

Comment:
In general, I prefer more `const`/immutable/`final`, but since I don't really know what's consuming this downstream all that well, I think I'll leave it like this...